src/client/features/view/cy/events/hover.js

```diff
@@ -174,4 +174,7 @@ const bindHover = (cy) => {
 
 };
 
-module.exports = bindHover;
\ No newline at end of file
+module.exports = {
+  bindHover,
+  storeStyle
```

Comment:
Do you still need storeStyle?

Reply:
No, storeStyle should have been removed form module.exports. It has now been removed.